BraveWallet/Crypto/Accounts/Add/AddAccountView.swift

```diff
@@ -17,8 +18,14 @@ struct AddAccountView: View {
   @State private var isLoadingFile: Bool = false
   @State private var originPassword: String = ""
   @State private var failedToImport: Bool = false
+  @State private var selectedCoin: BraveWallet.CoinType?
   @ScaledMetric(relativeTo: .body) private var privateKeyFieldHeight: CGFloat = 140.0
   @Environment(\.presentationMode) @Binding var presentationMode
+  
+  @ScaledMetric private var iconSize = 40.0
+  private let maxIconSize: CGFloat = 80.0
+  
+  var preSelectedCoin: BraveWallet.CoinType?
```

Comment:
This var isn't used at all?

Reply:
this is needed for stephen's network changes.  for example, user pick solana network but he doesn't have any solana account, we will present this view with `preselectedCoin = .sol`